package.json

```diff
@@ -98,6 +98,9 @@
         "@storybook/react": "6.5.16",
         "@storybook/storybook-deployer": "2.8.16",
         "@storybook/testing-library": "0.0.13",
+        "@testing-library/react": "12.1.5",
+        "@testing-library/jest-dom": "5.16.5",
+        "@types/react-test-renderer": "18.0.0",
```

Comment:
do we need [@testing-library/user-event](https://github.com/testing-library/user-event)?

Reply:
I used this MR as example http://tkgit-01.textkernel.net/searchbox/reach-ui/-/merge_requests/719/diffs we don't use it there. If we need it we can add when we will migrate all tests